scio-google-cloud-platform/src/main/java/com/spotify/scio/bigtable/ChannelPoolCreator.java

```diff
@@ -51,6 +51,6 @@ public static ChannelPool createPool(final BigtableOptions options) throws IOExc
     final ClientInterceptor[] interceptors = getClientInterceptors(options);
 
     return new ChannelPool(
-        () -> BigtableSession.createNettyChannel(options.getAdminHost(), options, interceptors), 1);
+        () -> BigtableSession.createNettyChannel(options.getAdminHost(), options, false, interceptors), 1);
```

Comment:
I set the `DirectPath` option to `false` bc. it was the default value before the upgrade. Note that I don't actually understand what this option actually does.  `BigtableSession.createNettyChannel` is an internal method and we should not be using it in the first place. On top of that `BigtableSession` has a method to directly create a `ChannelPool` so I don't understand why this class exists at all. @regadas would you knowwhy we have this ?

Reply:
Right, things are a bit fuzzy but I believe this was mainly created to support BT admin tasks. Haven't look at BigtableSession in awhile but it's possible that that method was not present when this was created. If it does exist and allows us to set the interceptors as well than we should replace this.